v2/en/community.md

```diff
@@ -4,6 +4,12 @@
     @created       2012-08-04 23:04 GMT
 --->
 
+# Official Forum
+
+We have established an official forum and imported the contents from mailing list (as of 2020-03-28). Welcome to visit:
```

Comment:
“established" => "set up"

"contents" => "archive"

"mailing list" => "mailing lists"

"2020-03-28" => "March 28, 2020"

"Welcome to visit" => "Welcome to join us there"